wolfcrypt/src/asn.c

```diff
@@ -17287,10 +17316,8 @@ static int DecodeAltNames(const byte* input, int sz, DecodedCert* cert)
                     }
                 }
 
-                /* test if no ':' char was found and test that the next two
-                 * chars are "//" to match the pattern "://" */
-                if (i >= strLen - 2 || (input[idx + i + 1] != '/' ||
-                                        input[idx + i + 2] != '/')) {
+                /* test if scheme is missing or heir-part is empty */
```

Comment:
We need a test case for the new feature.

Reply:
Yes. And, as I mentioned in the PR, I'm happy to write it if someone will generate a suitable certificate. Unfortunately, I wasn't able to follow the generation of test certificates in the library test suite to do it myself. 

Here's an example of how I generate the signing request through OpenSSL:
```
set DeviceId=9259c005-fa68-4857-b0e7-683bc4236f26
%OPENSSL% req -new -newkey rsa:4096 -keyout Output\Device-PrivateKey-%DeviceId%.pem ^
  -out Output\Device-CertificateRequest-%DeviceId%.pem ^
  -passout pass:secret ^
  -subj "/C=NZ/ST=Waikato/L=Hamilton/O=Blue Leaf Software Ltd/CN=Lumos/serialNumber=%DeviceId%" ^
  -addext "subjectAltName = URI:uuid:%DeviceId%" ^
  -addext "extendedKeyUsage = serverAuth, clientAuth"
```

Please let me know when you have a suitable certificate ready.